about/about.html

```diff
@@ -11,22 +11,29 @@
 
 <body>
     <header class="landing-page">
-        <div class="home-page">
-            <nav>
-                <div class="nav-bar">
-                    <img class="humbuger" src="./imagesabout/menu.png" alt="Menu" />
-                    <ul class="menu-item-hidden">
-                        <li class="cancel"><img src="./imagesabout/xicon.png" alt=""></li>
-                        <li><a href="../about/about.html" class="about">About</a></li>
-                        <li><a href="#Program" class="nav-list">Program</a></li>
-                        <li><a href="#join" class="nav-list">Join</a></li>
-                        <li><a href="#sponsor" class="nav-list">Sponsor</a></li>
-                        <li><a href="#news" class="nav-list">News</a></li>
-                        <li><a href="#campain" class="campain">CC Campain</a></li>
-                    </ul>
-                </div>
-            </nav>
-
+        <nav>
+        <div class="media">
+            <a href="#" class="twitter"><img src="./imagesabout/twitter.svg" alt="twitter" /></a>
+            <a href="#" class="linkedin"><img src="./imagesabout/linkdin.svg" alt="linkdin" /></a>
+            <a href="#" class="english">English</a>
+            <a href="#" class="page">My Page</a>
+            <a href="#" class="logout">Logout</a>
+        </div>
+    
+    <div class="nav-menu">
+            <img src="./imagesabout/logo.jpg" alt="logo"></a>
+        <ul class="menu-items">
+            <li class="cancel"><img src="./images/xicon.png" alt=""></li>
+            <li><a href="/index.html" class="home">Home</a></li>
```

Comment:
- [ ] (OPTIONAL) Good job adding the home link in the about page to fix navigation. Kindly arrange the `href` so that it takes the user to the home page